src/common/ZulipTextButton.js

```diff
@@ -76,6 +76,18 @@ type Props = $ReadOnly<{|
    */
   label: LocalizableReactText,
 
+  /**
+   * True just if the button is in the disabled state.
+   *   https://material.io/design/interaction/states.html#disabled
+   */
+  disabled?: boolean,
+
+  /**
+   * Whether `onPress` is used even when
+   *   `disabled` is true.
+   */
+  isPressHandledWhenDisabled?: boolean,
```

Comment:
formatting nit: give each jsdoc a summary line, as its own paragraph. Like this:

```diff
   /**
    * True just if the button is in the disabled state.
-   *   https://material.io/design/interaction/states.html#disabled
+   *
+   * https://material.io/design/interaction/states.html#disabled
    */
   disabled?: boolean,
 
   /**
-   * Whether `onPress` is used even when
-   *   `disabled` is true.
+   * Whether `onPress` is used even when `disabled` is true.
    */
   isPressHandledWhenDisabled?: boolean,
```